packages/storybook/CONTRIBUTING.md

```diff
@@ -1,12 +1,29 @@
-# Creating Storybook Component Documentation
+# Creating Storybook Files for a Component
```

Comment:
Can you remove the "Developer Workflows" section from the storybook readme and add a CONTRIBUTING section that links here?